src/components/RichTextInput.tsx

```diff
@@ -162,30 +164,68 @@ const formats = [
 	'emoji',
 ];
 
-function TextRichWithQuill(props: {
-	value: string;
-	setValue: any;
-	style: any;
-}) {
-	const { value, setValue, style } = props;
+function TextRichWithQuill(props: any) {
+	const [content, setContent] = useState('');
 	const [mod, setMod] = useState<any>();
 
 	useEffect(() => {
 		setMod(modules());
 	}, []);
 
+	useEffect(() => {
+		!mod && setMod(modules(props.projectId));
+	}, []);
+
+	useEffect(() => {
+		props.value && setContent(props.value);
+	}, [props.value]);
```

Comment:
In your suggested way the input won't recognize the change coming from the props - I changed this in particular to clean up the input, otherwise it wouldn't work

Reply:
Change in props.value doesn't make a rerender?

Reply:
No :( 
Not sure if it's because the empty string - But it doesn't, I tried that at first